tests/api/documents/test_documents_rest.py

```diff
@@ -180,35 +180,35 @@ def test_documents_facets(
     # FILTERS
     # person contribution
     list_url = url_for('invenio_records_rest.doc_list', view='global',
-                       contribution__de='Peter James')
+                       author='Peter James')
```

Comment:
Filtering authors with a language parameters seems not be tested.

Reply:
done